### ISSUE TITLE
How do I apply Total Row to last row in a table

### ISSUE THREAD

**Issue by muzzmatt** [python-openxml/python-docx#331]
I have been unable to find a way to apply the 'Total Row' style option (screenshot attached of the option in Word) to a table I have created. When selected within Word it applies special styling to the row similar to the Header row.

Is this possible?

![total_row_option](https://cloud.githubusercontent.com/assets/5155487/20099228/5e94208c-a5af-11e6-8d37-184026a57da4.png)

**Comment by scanny** [comment]
I don't believe that's supported by the API. I'll leave this open as a feature request for those special row formatting options.

**Comment by idahogray** [comment]
I would be willing to help add this feature if anyone is willing to mentor me. I am familiar with Python but not at all familiar with the docx format. I tried to compare two files, one with the total row enabled and one without but it wasn't immediately obvious how this works.

**Comment by telferm57** [comment]
It would be great to see this feature! Without it, I can't find a way of altering column banding  .. the default table style options seem to override the settings in the table style itself

**Comment by scanny** [comment]
Closed as inactive.
